src/components/About.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import experience from './workExperience.json'
+
+export default function About() {
+  return (
+    <main className='flex flex-col justify-start items-start gap-5 3xl:py-[5rem] xl:py-[5rem] xl:px-0 md:py-[2rem] md:px-10 sm:py-[5rem] sm:px-5 '>
+      <h1 className='xl:text-5xl md:text-4xl sm:text-3xl dark:text-white text-[#42446E] capitalize '>
+        about me
+      </h1>
+      <p className=''>
+        I am a seasoned front-end developer with a rich background in crafting
+        and nurturing web applications. My journey is marked by mastery of an
+        array of cutting-edge technologies, including{' '}
+        <strong>
+          React, JavaScript, BootStrap, SASS, TypeScript, VS Code, Git & GitHub,
+          Tailwind, Next.js, CSS, HTML, and Redux Toolkit.
+        </strong>{' '}
+        My skills are not just a reflection of my experience; they are a
+        testament to my unwavering commitment to staying at the forefront of web
+        development.
+      </p>
+      <p className=''>
+        Collaboration is my second nature, and my thirst for knowledge is
+        unquenchable. I thrive in dynamic, forward-thinking environments, where
+        innovation is the driving force. My true passion lies in the art of
+        fashioning user-friendly, visually captivating web applications that
+        resonate with users.
+      </p>
+      <p className=''>
+        At this juncture, I am fervently seeking a Frontend Developer role that
+        aligns with my aspirations in a high-octane, forward-looking setting. I
+        hold the firm belief that I can bring a transformative influence to your
+        team, elevating your projects to new heights. Let&apos;s embark on this
```

Comment:
I fixed the " sign bug here, hence I need to pay attention to this moving forward.